WIP: mdr robocup tasks/find my mates

PatrickNa opened the pull request:
The aim is to define the "find my mates" scenario for the RoboCup@Home competition. This pull request addresses issue #105

Commit:
Modify the launch files of question_answering andd speech_matching.

With this commit the launch files include nodes that are required by the node to run properly. E.g. mdr_question_answering now also executes mdr_speech_recognition and mdr_speech_matching.

Commit:
Add the structure for the "find my mate" scenario.

This commit provides the pkg structure for the "find my mate" task of the RoboCup@Home competition. The state machine and launch file are not defined, yet. This will be addressed in the upcoming commits.

minhnh commented on mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_find_my_mates/CMakeLists.txt:
```diff
@@ -0,0 +1,195 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mdr_find_my_mates)
+
+## Compile as C++11, supported in ROS Kinetic and newer
```
@argenos @alex-mitrevski what's the consensus for removing the auto-generated comments in  `CMakeLists.txt`? I personally prefer to remove them so the file is not too cluttered.

robertocaiwu replied:
I'm also in favor of removing it.

alex-mitrevski replied:
I always remove those.

PatrickNa replied:
Will do at some point.

Commit:
Add description and best case of the scenario.

This commit provides a description of the task, the sub-tasks, the goal and restrictions. Addtionally, a first state machine of the scenario is defined. The state machine focuses on one loop only and covers the best case of the scenario. In following commits further adjustments to the state machine need to be done.

Commit:
Include the second loop within the scenario and failing use case.

This commit provides a more elaborated state machine of the scenario. It includes the second loop withing the scenario (search and analysis of a second guest) and the exit transitions in case of failure.

PatrickNa commented:
The following figure visualizes the scenario up to this point:
![find_my_mates](https://user-images.githubusercontent.com/17699225/54083196-c551d600-4320-11e9-997a-5316e0264df3.png)

It can be noted that some of the actions are related with each other. For example the _request_actions can most probably originate from the same action with a different content.

The action _Localize Person_ includes the following sub-processes:
- Navigate around the area.
- Scan the surroundings looking for persons (#26 ).
- Save the location of the person.

The actions _Analyze Location_ and _Analyze Person_ refer to perception and the recognition of identifying objects.
